src/util.rs

```diff
@@ -261,9 +268,18 @@ impl AdsMessage {
                     } else {
                         debug!("{prefix}: no error invoke_id={invoke_id}\n");
                     }
+                    debug_printed = true;
                 }
             }
         }
+        if ! debug_printed {
+            let prefix = format!("{in_out_bh_clnt:?} {} {}:{}[{inv:#08x}]->{}:{}  {cmdname}",
```

Comment:
Why the two step formatting? Just `debug!("{in_out_bh_clnt:?} ...", ...)` should be better.

Reply:
Thanks for the review - should be fixed now